crates/cheatnet/src/cheatcodes/mod.rs

```diff
@@ -17,6 +17,7 @@ pub mod mock_call;
 pub mod prank;
 pub mod precalculate_address;
 pub mod roll;
+pub mod signing;
```

Comment:
```suggestion
pub mod signature;
```
For consistency with rust please create signature module keeping the general traits.
Please create module with specific crypto primitive name and keep the implementations of this primitive there.